src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration/Options.cs

```diff
@@ -24,26 +25,49 @@ public enum SettingFields : uint
         All = uint.MaxValue
     }
 
-    public class RequestOptions
+    public class ConfigurationSelector
     {
+        public static readonly string Any = "*";
         /// <summary>
-        /// Specific label of the key.
+        /// Keys that will be used to filter.
         /// </summary>
-        public string Label { get; set; } = null;
-
-        public ETagFilter ETag { get; set; }
-
+        /// <remarks>See the documentation for this SDK for details on the format of filter expressions</remarks>
+        public List<string> Keys { get; set; } = new List<string> { Any };
```

Comment:
I think this should return IList<string>. It would allow us to optimize the storage for the strings if we ever need to.